core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -223,11 +225,11 @@ object ConsumerGroupCommand {
                 .format(group, topicAndPartition))
           }
         }
-        else if (offsetAndMetadata.error == ErrorMapping.NoError)
+        else if (offsetAndMetadata.error == Errors.NONE.code)
```

Comment:
This is a result of the old Requests and Responses which I don't want to change and will be removed going forward. The new o.a.k Requests and Response already do this. I have a refactoring patch coming soon to remove as much of the old Request and Response stuff as possible.